src/test/scala/dotty/xml/interpolator/ReporterTest.scala

```diff
@@ -57,83 +57,83 @@ class ReporterTest {
   }
 
   @Test def attribute1(): Unit = {
-    assertEquals(xml"<foo bar", List((8, "'=' expected but end of source found")))
+    assertEquals(xml"<foo bar", List((5, "'>' expected but 'b' found")))
```

Comment:
Why are the outputs different?

Reply:
Are the parser combinators behaving differently in `"scala-parser-combinators" % "1.1.2"`?

Reply:
Or `"scala-xml" % "1.2.0"` changed something?

Reply:
It looks like the scala-parser-combinators version change affected the errors?

Reply:
It looks ok